src/usecases/retrieveHospitalVisitTotals.contractTest.js

```diff
@@ -0,0 +1,83 @@
+import AppContainer from "../containers/AppContainer";
+
+describe("retrieveHospitalVisitTotals contract tests", () => {
+  const container = AppContainer.getInstance();
+
+  it("returns the total number of visits for each hospital", async () => {
+    const { trustId } = await container.getCreateTrust(container)({
```

Comment:
Do we need the `container` to be passed into `getCreateTrust`?

Reply:
good catch thanks